lib/cublas/wrappers.jl

```diff
@@ -133,6 +133,9 @@ for (jname, fname, elty) in ((:dot,:cublasDdot_v2,:Float64),
         end
     end
 end
+dot(n::Integer, x::DenseCuArray{Float16}, y::DenseCuArray{Float16}) = convert(Float16, dot(n, convert(DenseCuArray{Float32}, x), convert(DenseCuArray{Float32}, y)))
```

Comment:
Maybe add a TODO here; `cublasDotEx` _does_ support F16.

Reply:
Oh cool, I can just use that instead on CUDA versions which support it, then.